upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -407,6 +466,15 @@ spec:
           spec:
             description: FelixConfigurationSpec contains the values of the Felix configuration.
             properties:
+              awsSrcDstCheck:
+                description: 'Set source-destination-check on AWS EC2 instances. Accepted
+                  value must be one of "DoNothing", "Enabled" or "Disabled". [Default:
+                  DoNothing]'
+                enum:
+                - DoNothing
+                - Enable
+                - Disable
+                type: string
```

Comment:
Do we want to move the logic from https://github.com/kubernetes/kops/blob/fc2e70e1e8e8901b6713ac930dded00dff077516/upup/models/bindata.go#L8618 for installing the `k8s-ec2-srcdst` pod to here as this is now the supported path?

I think allowing users to enable this will require them to add permissions to worker nodes as we currently only give the required `ec2:ModifyInstanceAttribute` permissions to masters, at a minimum I think we need to add something to the docs, and make clear that enabling this alongside the existing logic for installing `k8s-ec2-srcdst` is an untested path.

Reply:
Yes, we do, and also enable WireGuard. But best to do it in some follow-up PRs.

Reply:
Does _calico-node_ talk to the AWS API for this, or is it the _calico-kube-controllers_ pod? If it's the latter, we could optionally confine it to run only on master nodes, and grant the IAM permissions only to the instance profile used by the masters.

_[Time passes...]_

[It's part of Felix,](https://github.com/projectcalico/felix/blob/master/aws/ec2.go) and therefore [part of _calico-node_](https://docs.projectcalico.org/reference/node/configuration).